packages/custom_lint/lib/src/workspace.dart

```diff
@@ -424,6 +424,11 @@ Iterable<String> _findRoots(String path) sync* {
   final directory = Directory(path);
 
   yield* directory.listSync(recursive: true).whereType<File>().where((file) {
+    // Skip the root of dependency caches.
+    if (file.path.contains('.dart_tool') || file.path.contains('.symlinks')) {
```

Comment:
That's valuable, but this isn't the right fix. We should instead ignore pubspecs without a package_config.json

Reply:
So, is this change correct? https://github.com/invertase/dart_custom_lint/pull/269/commits/fd024f6a8849113789bcf793cba3c870785cc616
(I thought that maybe we should check it after return `_findRoots`.)